Remove information overload from toctrees (#409)
NixOS/nix.dev: source/tutorials/index.md

# Tutorials

> This section contains a collection of articles on how to achieve a goal with the Nix ecosystem.

Reviewer: ```suggestion
This section contains a collection of articles on how to get thing done in the Nix ecosystem.
```

Reviewer 2: s/thing/things

Reviewer: Feel free to ninja-edit my typo-ridden suggestions.
